Lesson-4-3

jocl0110 opened the pull request.

jocl0110 committed:
Add a form element to let the user leave a message

jocl0110 committed:
Configure the form to print the messages in the website and remove them

jocl0110 committed:
Add the reset method cause I accidentally deleted it

Magale commented:
Hi @jocl0110, you did a good job once gain. Keep it up!

Another way you could have used to get text from the form is the "messageForm.elements.elementName.value" method.
See example below of how to get the user name from the form;

const usersName = messageForm.elements.usersName.value;

The getElementsByName method is not commonly used for form fields like input, textarea, etc

I will email you the entire JavaScript code I modified.

jocl0110 commented:
> Hi @jocl0110, you did a good job once gain. Keep it up!
> 
> Another way you could have used to get text from the form is the "messageForm.elements.elementName.value" method. See example below of how to get the user name from the form;
> 
> const usersName = messageForm.elements.usersName.value;
> 
> The getElementsByName method is not commonly used for form fields like input, textarea, etc
> 
> I will email you the entire JavaScript code I modified.



> Hi @jocl0110, you did a good job once gain. Keep it up!
> 
> Another way you could have used to get text from the form is the "messageForm.elements.elementName.value" method. See example below of how to get the user name from the form;
> 
> const usersName = messageForm.elements.usersName.value;
> 
> The getElementsByName method is not commonly used for form fields like input, textarea, etc
> 
> I will email you the entire JavaScript code I modified.

Hello Magale. Good Afternoon. I not very sure if you will see this message as a reply cause this is my first time answering messages here. Thanks for your advices. King Regards.

Magale commented:
Hi @jocl0110 , I saw your message.
